conf/syslog.conf.defaults

```diff
@@ -0,0 +1,3 @@
+[local]
+type=file
+logs=collectd.log,fingerbank.log,httpd.aaa.access,httpd.aaa.error,httpd.admin.access,httpd.admin.catalyst,httpd.admin.error,httpd.admin.log,httpd.collector.error,httpd.collector.log,httpd.graphite.access,httpd.graphite.error,httpd.parking.access,httpd.parking.error,httpd.portal.access,httpd.portal.catalyst,httpd.portal.error,httpd.proxy.access,httpd.proxy.error,httpd.webservices.access,httpd.webservices.error,packetfence.log,pfbandwidthd.log,pfconfig.log,pfdetect.log,pfdhcplistener.log,pfdns.log,pffilter.log,pfmon.log,radius-acct.log,radius-cli.log,radius-eduroam.log,radius-load_balancer.log,radius.log,redis_cache.log,redis_ntlm_cache.log,redis_queue.log
```

Comment:
If its easy to do, an ALL keyword would be nice or an additional parameter for the section to send them all.

Would make it easier to configure and then when we would add new logs in upgrade, if the user selected the 'all' option, they would automagically be sent